doc/manuals-ld/installation-guide-centos7.md

```diff
@@ -17,7 +31,9 @@ To be installed via package manager:
 Tools needed for compilation and testing:
 
 ```bash
-sudo yum install make cmake gcc-c++ scons
+sudo yum install http://repo.okay.com.mx/centos/7/x86_64/release/okay-release-1-1.noarch.rpm
```

Comment:
What is this?   okay, from Mexico?

People will wonder.
Add a short description on why this package is necessary